test/e2e/common/client.go

```diff
@@ -6,8 +6,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 
+	admissionv1 "k8s.io/api/admission/v1"
```

Comment:
Do we really need to keep admissionv1beta1 in the line below?

Reply:
It seems to be the other way around. I can't figure out where the v1beta1 scheme is getting registered in the operator, but apparently it is, because the error if I keep both v1 and v1beta1 in `admissionReviewVersions` in the webhook configs indicates that the v1 scheme isn't registered; but if I take it out I don't get that complaint. My theory was that I need to register both in order to list both in the configs.

Anyway, I'm going to try to remove v1 here and in the configs and see if that works.